front/app/modules/commercial/machine_translations/hooks/useTranslation.ts

```diff
@@ -10,13 +10,15 @@ interface Parameters {
   localeTo?: SupportedLocale;
   id: string;
   context: 'idea' | 'initiative' | 'comment';
+  machineTranslationButtonClicked: boolean;
 }
 
 export default function useTranslation({
   attributeName,
   localeTo,
   id,
   context,
+  machineTranslationButtonClicked,
```

Comment:
Does this fix the issue where lots of requests were happening? Perhaps we could add a comment or so, so someone doesn't decide to remove this (as this would cause a big bill for services at Google)?